cli/src/main/java/de/jplag/cli/CliOptions.java

```diff
@@ -0,0 +1,142 @@
+package de.jplag.cli;
+
+import java.io.File;
+
+import de.jplag.Language;
+import de.jplag.clustering.ClusteringAlgorithm;
+import de.jplag.clustering.ClusteringOptions;
+import de.jplag.clustering.algorithm.InterClusterSimilarity;
+import de.jplag.options.JPlagOptions;
+import de.jplag.options.SimilarityMetric;
+
+import picocli.CommandLine;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@CommandLine.Command(name = "jplag", description = "", usageHelpAutoWidth = true, abbreviateSynopsis = true)
```

Comment:
I meant examples like --bc and -bc, where the keyword is identical, but the only difference is the dashes. If we do not go the enum route, leave it as is for now. With the enum route, this can be cleanly deduplicated. However, these are fine details I would say.